queries.yaml

```diff
@@ -881,6 +881,73 @@ post_process:
         (kind == 'rail' and zoom < 15) or
         (kind == 'minor_road' and zoom < 14) or
         (kind == 'major_road' and zoom <  7)
+  # drop these "detail" tags to get better merging at zoom < 16
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: roads
+      start_zoom: 0
+      end_zoom: 16
+      properties:
+        - colour
+        - motor_vehicle
+        - operator
+        - route
+        - route_name
+        - state
+        - symbol
+        - type
+      where: >-
+        kind == 'minor_road'
+  # keep "surface" until zoom 15.
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: roads
+      start_zoom: 0
+      end_zoom: 15
+      properties:
+        - surface
```

Comment:
It looks like surface is used to set "brown" color on line and text halo in Walkabout at some zooms, but looks like only for `kind` = **path** so we're good?

https://github.com/tangrams/walkabout-style/blob/gh-pages/layers/bike-interlay.yaml#L705-L708